docs/book/introduction.md

```diff
@@ -16,15 +16,135 @@ description: Welcome to ZenML!
 
 If you answered yes to any of the questions above, ZenML is here to help you:
 
-* **ZenML** is an **open-source MLOps framework** built for data scientists, ML Engineers, and MLOps Developers.
-* It comes equipped with a simple and flexible **Python SDK** that helps you to create **portable and production-ready pipelines**.
-* It also features **cloud-** and **tool-agnostic** **interfaces/abstractions** such as orchestration systems, artifact repositories, and model deployment platforms that cover the entire ML journey.
-* With these abstractions, it remains **extensible** and **open to customization**. That is also backed by numerous **collaborative features** that allow you to work not just with your team but also with the community.
-* Moreover, this creates a separation between the different aspects of ML workflows, allowing users to focus on creating their own respective solutions in an efficient manner.
-* Overall, it reduces the complexity of your development lifecycle, all the way **from an experimentation environment to a production-grade solution**.
+**ZenML** is an extensible, open-source MLOps framework for creating portable, 
+production-ready MLOps pipelines. 
+
+By decoupling infrastructure from code, ZenML gives all ML developers in your 
+organization more freedom and independence in how they approach their work:
+
+![ZenML Overview](.gitbook/assets/intro_zenml_overview.png)
+
+{% tabs %}
+
+{% tab title="ZenML for Data Scientists" %}
+
+As a Data Scientists, ZenML gives you the freedom to focus on modeling and
+experimentation without having to worry about production issues at all.
```

Comment:
Is it 'you don't have to worry about production', or is it 'we empower you that you can get to do things that otherwise you wouldn't have been able to do'?

Reply:
I also agree that we should rather focus that we connect you to production

Reply:
Good point, current formulation is a bit too negative perhaps 👍